app/src/main/java/com/example/poste/activities/EditPostActivity.java

```diff
@@ -113,6 +112,7 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                             currentPost.setDescription(description);
                             currentPost.setUrl(url);
                             Toast.makeText(EditPostActivity.this,"Edit post successful!", Toast.LENGTH_SHORT).show();
+                            finish();
```

Comment:
Would suggest placing this after the if/else for response.isSuccessful, that way it returns on either case; onFailure as well

Reply:
So my though for using the finish within the successful block only was that if the edit failed for some reason and the user would like to try again. If we finish even in case of a failure, the changes that the user made will be lost. 
The control flow in my head was that if the edit failed, and the user would like to exit, they could click cancel changes and that would finish the activity.

As for the commented lines, I can go ahead and remove them to reduce confusion.

Reply:
To avoid merge conflict with the big US185 branch, is it acceptable to you to include these changes in the US185 PR and close this one?

Reply:
I see you have included it already. So I will go ahead and close out this PR.